core/src/test/java/greencity/controller/PlaceControllerTest.java

```diff
@@ -428,28 +429,6 @@ void getFilteredPlaces() throws Exception {
         verify(placeService).getPlacesByFilter(filterPlaceDto, userVO);
     }
 
-    @Test
-    void updateStatus() throws Exception {
```

Comment:
why did you remove this test?

Reply:
The task involved fixing the old code, so I modified the existing method to ensure it sends an email to the user and saves valid data in the database. Since the test was written for the method I modified, I deleted the old test and added new tests methods to cover the updated functionality.